apps/src/javalab/PlaygroundVisualizationColumn.jsx

```diff
@@ -79,7 +99,9 @@ const styles = {
 export default connect(
   state => ({
     isReadOnlyWorkspace: state.pageConstants.isReadOnlyWorkspace,
-    isCollapsed: state.javalab.isVisualizationCollapsed
+    isCollapsed: state.javalab.isVisualizationCollapsed,
+    playgroundItemData: state.javalab.playgroundItemData,
```

Comment:
(maybe a follow-up) Should we split out a redux for playground specifically?

Reply:
Sanchit had the same thought, going to do that